plugins/deprecation.nix

```diff
@@ -10,13 +10,31 @@ let
       - `:PreviewQuery` to open the Query Editor (Nvim 0.10+)
     '';
   };
+  renamed = {
+    # Added 2024-09-17
+    surround = "vim-surround";
+  };
 in
 {
-  imports = lib.mapAttrsToList (
-    name:
-    lib.mkRemovedOptionModule [
-      "plugins"
-      name
-    ]
-  ) removed;
+
+  imports =
+    (lib.mapAttrsToList (
+      name:
+      lib.mkRemovedOptionModule [
+        "plugins"
+        name
+      ]
+    ) removed)
+    ++ (lib.mapAttrsToList (
+      oldName: newName:
```

Comment:
I think `old` and `new` is fine, the added verbosity of `*Name` shouldn't be needed given the surrounding context.